common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java

```diff
@@ -0,0 +1,327 @@
+        Assert.assertNull(UtilAll.string2bytes(null));
+
+        Assert.assertArrayEquals(new byte[]{-1}, UtilAll.string2bytes("foo"));
+    }
+
+    @Test
+    public void testUncompress() throws IOException {
+        byte[] by = new byte[]{120, 1, 1, 3, 0, -4, -1, 1, 2, 3, 0, 13, 0, 7};
+        Assert.assertArrayEquals(new byte[]{1, 2, 3}, UtilAll.uncompress(by));
+    }
+
+    @Test
+    public void testCompress() throws IOException {
+        byte[] by = new byte[]{120, 1, 1, 3, 0, -4, -1, 1, 2, 3, 0, 13, 0, 7};
+        Assert.assertArrayEquals(by, UtilAll.compress(new byte[]{1, 2, 3}, 0));
+    }
+
+    @Test
+    public void testAsInt() {
+        Assert.assertEquals(1, UtilAll.asInt("foo", 1));
+        Assert.assertEquals(123, UtilAll.asInt("123", 1));
+    }
+
+    @Test
+    public void testAsLong() {
+        Assert.assertEquals(1l, UtilAll.asLong("foo", 1l));
+        Assert.assertEquals(1556195014000l,
+                UtilAll.asLong("1556195014000", 1l));
+    }
+
+    @Test
+    public void testFormatDate() {
+        Assert.assertEquals("2019-04-25 13:23:34", UtilAll
+                .formatDate(new Date(1556195014000l), "yyyy-MM-dd HH:mm:ss"));
+    }
+
+    @Test
+    public void testParseDate() {
+        Assert.assertNull(UtilAll.parseDate("2019-04-25 13:23:34", ""));
+
+        Assert.assertEquals(new Date(1556195014000l), UtilAll
+                .parseDate("2019-04-25 13:23:34", "yyyy-MM-dd HH:mm:ss"));
+    }
+
+    @Test
+    public void testResponseCode2String() {
+        Assert.assertEquals("2", UtilAll.responseCode2String(2));
+    }
+
+    @Test
+    public void testFrontStringAtLeast() {
+        Assert.assertEquals("", UtilAll.frontStringAtLeast("", 2));
+        Assert.assertEquals("fo", UtilAll.frontStringAtLeast("foo", 2));
+    }
+
+    @Test
+    public void testIsBlank() {
+        Assert.assertTrue(UtilAll.isBlank(""));
+        Assert.assertTrue(UtilAll.isBlank(" "));
+        Assert.assertTrue(UtilAll.isBlank(null));
+
+        Assert.assertFalse(UtilAll.isBlank("foo"));
+    }
+
+    @Test
+    public void testJstack() {
+        Assert.assertNotNull(UtilAll.jstack());
+    }
+
+    @Test
+    public void testIsInternalIP() {
+        Assert.assertFalse(UtilAll.isInternalIP(new byte[]{1, 1, 1, 1}));
+
+        Assert.assertTrue(UtilAll.isInternalIP(new byte[]{10, 1, 2, 3}));
+        Assert.assertTrue(UtilAll.isInternalIP(
+                new byte[]{(byte) 172, 16, 2, 3}));
+        Assert.assertTrue(UtilAll.isInternalIP(
+                new byte[]{(byte) 192, (byte) 168, 2, 3}));
+    }
+
+    @Test
+    public void testIpToIPv4Str() {
+        Assert.assertEquals("10.1.2.3",
+                UtilAll.ipToIPv4Str(new byte[]{10, 1, 2, 3}));
+        Assert.assertEquals("172.16.2.3",
+                UtilAll.ipToIPv4Str(new byte[]{(byte) 172, 16, 2, 3}));
+        Assert.assertEquals("192.168.2.3",
+                UtilAll.ipToIPv4Str(new byte[]{(byte) 192, (byte) 168, 2, 3}));
```

Comment:
line > 80 chars (81)

Reply:
It is 80 chars for me.. I moved it tho

Reply:
Sorry, I double-checked and you were right.